### ISSUE TITLE
Updating a sector state does not work

### ISSUE THREAD

**Issue by RobQuistNL** [filecoin-project/lotus#8697]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [ ] I did not make any code changes to lotus.

### Lotus component

- [ ] lotus daemon - chain sync
- [ ] lotus miner - mining and block production
- [X] lotus miner/worker - sealing
- [X] lotus miner - proving(WindowPoSt)
- [ ] lotus miner/market - storage deal
- [ ] lotus miner/market - retrieval deal
- [ ] lotus miner/market - data transfer
- [ ] lotus client
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [ ] Other

### Lotus Version

```text
1.15.3-rc1
```

### Describe the Bug

```
$ lotus-miner sectors list
481  Committing  NO       NO      n/a                           CC
```

```
$ lotus-miner sectors update-state --really-do-it 481 ComputeProofFailed

```

```
$ lotus-miner sectors list
481  Committing  NO       NO      n/a                           CC
```

```
$ lotus-miner sectors update-state --really-do-it 481 SubmitCommitAggregate

```

```
$ lotus-miner sectors list
481  Committing  NO       NO      n/a                           CC
```

### Logging Information

```text
-
```

### Repo Steps

Run update-state command
See that it does not do anything

**Comment by RobQuistNL** [comment]
Last log:

```
10.	2022-05-22 23:33:01 +0000 UTC:	[event;sealing.SectorFinalizedAvailable]	{"User":{}}
11.	2022-05-22 23:33:01 +0000 UTC:	[event;sealing.SectorSubmitCommitAggregate]	{"User":{}}
12.	2022-05-22 23:34:01 +0000 UTC:	[event;sealing.SectorCommitFailed]	{"User":{}}
	sector failed in aggregate processing: pushing message to mpool: GasEstimateMessageGas error: estimating gas used: message execution failed: exit SysErrInsufficientFunds(6), reason: failed to transfer funds (RetCode=6): transfer failed, insufficient balance in sender actor: 129215709719129130 (RetCode=6)
13.	2022-05-22 23:35:01 +0000 UTC:	[event;sealing.SectorRetryComputeProof]	{"User":{}}
```

Added all the funds but its not moving anymore.

**Comment by vanvank** [comment]
I also have this issue

**Comment by Reiers** [comment]
Do you still have this issue? @RobQuistNL 
I can update-state here on:
```
lotus version
Daemon:  1.17.0-rc3+mainnet+git.9794652e0+api1.5.0
Local: lotus version 1.17.0-rc3+mainnet+git.9794652e0
```

**Comment by RobQuistNL** [comment]
I'm not upgrading to RC versions ever again, we'll know once 1.17.0 hits stable :)

**Comment by RobQuistNL** [comment]
??? how about lets wait until 1.17.0 hits stable so I can test it

**Comment by RobQuistNL** [comment]
Fixed with 1.17.0 :partying_face:
